.travis.yml

```diff
@@ -48,4 +51,19 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-    - $HOME/.android/build-cache
\ No newline at end of file
+    - $HOME/.android/build-cache
+
+deploy:
```

Comment:
Unfortunately we cannot.  This is because travis does not support secrets in PRs.  Probably doable with Jenkins but not with any of the cloud based CI systems like Travis or Circle CI.

Reply:
I don't mean just PRs. Running tests prior to deploy applies even more on a release branch. We shouldn't deploy to play store prior to Test Lab

Reply:
ok, i see what you mean.  i guess we would need to setup a separate `firebase-test` stage that would only runs on non-PR builds.

Reply:
done